treesit-auto.el

```diff
@@ -112,6 +112,16 @@ automatic installation (or prompting, based on the value of
       :ts-mode 'clojure-ts-mode
       :remap 'clojure-mode
       :url "https://github.com/sogaiu/tree-sitter-clojure")
+    ,(make-treesit-auto-recipe
+      :lang 'clojure
+      :ts-mode 'clojurec-ts-mode
+      :remap 'clojurec-mode
+      :url "https://github.com/sogaiu/tree-sitter-clojure")
```

Comment:
Does clojurc use a special extension?  I'm not familiar with the language, so I'd need your guidance here.  I'm not sure if this would clash with the `:ext "\\.clj\\'"` in the line above.